website/content/docs/discovery/dns.mdx

```diff
@@ -52,7 +52,7 @@ There are fundamentally two types of queries: node lookups and service lookups.
 A node lookup, a simple query for the address of a named node, looks like this:
 
 ```text
-<node>.node[.datacenter].<domain>
+<node>.node[.<datacenter>].<domain>
```

Comment:
Most of these small DNS docs improvements (e.g., clarifying which values are variables by wrapping with `<>`) will be moved into a separate PR.

Reply:
Refer to https://github.com/hashicorp/consul/pull/13999